### DISCUSSION TITLE
Reload fzf colors on colorscheme change

### DISCUSSION THREAD

**Discussion by sanchay9** [ibhagwan/fzf-lua#1428] [Q&A]
I often end up like this:

![image](https://github.com/user-attachments/assets/a915b5af-5069-4681-a94d-8c544ab8ce40)

is there any way we can reload the fzf window colors to match with the colorscheme

**Comment by ibhagwan** [comment]
That’s why you have `:FzfLua setup_highlights`, create an autocmd that runs that on `ColorScheme` event.
